lib/msal-common/src/request/RefreshTokenRequest.ts

```diff
@@ -14,4 +14,5 @@ import { BaseAuthRequest } from "./BaseAuthRequest";
  */
 export type RefreshTokenRequest = BaseAuthRequest & {
     refreshToken: string;
+    claims?: string;
```

Comment:
Should we make this type `object` in the Request objects and stringify when adding it to the request?